router/validation.go

```
@@ -0,0 +1,177 @@
+		}
+
+		if err := validateJSONBody(body); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// validateBulkRequest validates a bulk API request
+func validateBulkRequest(r *http.Request) error {
```

golangci-lint (1.62.2)

func `validateBulkRequest` is unused

(unused)